test/distributed/checkpoint/test_state_dict.py

```diff
@@ -301,7 +306,7 @@ def test_ddp(self) -> None:
         self.run_subtests(
             {
                 "use_composable": [True, False],
-                "optimizer_class": [torch.optim.Adam, torch.optim.AdamW],
+                "optimizer_class": [torch.optim.SGD],
```

Comment:
Do we want to add torch.optim.SGD to the original list instead?

Reply:
good catch